apps/web-giddh/src/app/settings/financial-year/financial-year.component.ts

```diff
@@ -101,28 +101,29 @@ export class FinancialYearComponent implements OnInit, OnDestroy {
                     }
                 });
                 this.yearOptions = cloneDeep(yearOptions);
-                this.forceClear$ = observableOf({ status: true });
+                // this.forceClear$ = observableOf({ status: true });
```

Comment:
revert this code and if it is unused then remove variable

Reply:
done